src/map/ai/helpers/targetfind.cpp

```diff
@@ -187,8 +195,8 @@ void CTargetFind::findWithinCone(CBattleEntity* PTarget, float distance, float a
     m_CPoint.x = cosf((2 * (float)M_PI) - leftAngle) * distance + m_APoint->x;
     m_CPoint.z = sinf((2 * (float)M_PI) - leftAngle) * distance + m_APoint->z;
 
-    // ShowDebug("angle %f, rotation %f, distance %f, A (%f, %f) B (%f, %f) C (%f, %f)\n", angle, rightAngle, distance, m_APoint->x, m_APoint->z, m_BPoint.x, m_BPoint.z, m_CPoint.x, m_CPoint.z);
-    // ShowDebug("Target: (%f, %f)\n", PTarget->loc.p.x, PTarget->loc.p.z);
+    //ShowDebug("angle %f, rotation %f, distance %f, A (%f, %f) B (%f, %f) C (%f, %f)\n", angle, rightAngle, distance, m_APoint->x, m_APoint->z, m_BPoint.x, m_BPoint.z, m_CPoint.x, m_CPoint.z);
```

Comment:
why are you removing the leading space here? `// stuff` actually helps my eyes (I have rather poor eyesight) so I habitually write mine that way.

Reply:
I assume that anything that is code and flagged out should be left with no space? I thought that was a convention used in some code to differentiate toggle on/off code from actual comments?

Reply:
nope just different people editing before we had any style and a few after where its not really enforced. a lot of "quick comment this out" commits have 3 or more different ways of doing it:
```cpp
//                          thing
```
```cpp
//thing
```
```cpp
// thing
```
my preference is that last one. This happens in the lua scripts to, but with -- instead of //